runtimes/rocm-pytorch/ubi9-python-3.9/utils/bootstrapper.py

```diff
@@ -750,8 +750,7 @@ def main():
     input_params = OpUtil.parse_arguments(sys.argv[1:])
     OpUtil.log_operation_info("starting operation")
     t0 = time.time()
-    OpUtil.package_install(user_volume_path=input_params.get("user-volume-path"))
-
+    
```

Comment:
Can we get rid of the trailing spaces? :angel:

Reply:
Done